mod-source-record-manager-server/src/main/java/org/folio/services/ChangeEngineServiceImpl.java

```diff
@@ -267,6 +267,13 @@ private boolean updateInstanceActionExists(JobExecution jobExecution) {
       Action.UPDATE);
   }
 
+  private boolean isCreateItemActionExists(JobExecution jobExecution) {
```

Comment:
This condition should be wider - we need to skip the "save records in SRS" part for profiles that
**contain Create/Update actions on Holdings/Item, but no Create Instance**